hpe3parclient/client.py

```diff
@@ -4130,3 +4130,48 @@ def getSnapshotsOfVolume(self, cpgName, volName):
                 if volume['copyOf'] == volName:
                     snapshots.append(volume['name'])
         return snapshots
+
+    def createSchedule(self, schedule_name, volume_name, tasksch, snaprefix='sv-', expiration = None, retain = None, readonly=False):
+        """Create Schedule for volume snapshot.
+        :param schedule_name - The name of the schedule
+        :type - string
+        :param volume_name - The name of the volume
+        :type - string
+        :param expiration - Expiration period for snapshot
+        :type - string
+        :retain - Retaintion period for snapshot
+        :type - string
+        :taskschedule - schedule for snapshot created
+        :type - string
+        """
+        cmd = ['createsched "createsv']
+        if readonly:
+            cmd.append['-ro']
+        if expiration != None:
+            cmd.append('-exp')
+            cmd.append(expiration)
+        if retain != None:
+            cmd.append('-retain')
+            cmd.append(retain)
+        cmd.append(snaprefix+volume_name)
+        cmd.append(volume_name+"\"")
+        if '@' not in tasksch:
+           cmd.append("\""+tasksch+"\"")
+        else:
+           cmd.append(tasksch)         
+        cmd.append(schedule_name)
+        #self._run(cmd)
+        
+        #self._run(cmd)
+        command = ' '.join(cmd)
+        self.ssh.open()        
+        self.ssh._ssh_execute(command, check_exit_code=True)
```

Comment:
How do we know if the command passed or failed? Wrap it under try except. If the result of the command contains FAILED or INVALID, throw an exception